js/gulpfile.js

```diff
@@ -116,17 +110,6 @@ gulp.task("dist", gulp.parallel(libs.map(libName => libTask(libName, "generate")
 
 gulp.task("dist:clean", gulp.parallel(libs.map(libName => libTask(libName, "clean"))));
 
-gulp.task("ice:module:package", () => gulp.src(["package.json"]).pipe(gulp.dest("node_modules/@zeroc/ice")));
```

Comment:
This is not required with the current test setup. Previously we have to copy sources into node_modules to make it visible to the test suite.